src/js/calculator.js

```diff
@@ -0,0 +1,102 @@
+export class Calculator {
+  constructor() {
+    this.firstNumberString = "";
+    this.secondNumberString = "";
+    this.operator = "";
+    this.displayText = "";
+    this.hasClickedOperator = false;
+  }
+
+  setEventListeners() {
+    const digits = document.getElementsByClassName("digit");
+    for (const digit of digits) {
+      digit.addEventListener("click", () => {
+        if (this.hasClickedOperator) {
+          if (this.isLessThanThreeDigits(this.secondNumberString)) {
+            this.secondNumberString += digit.textContent;
+            this.displayText =
+              this.firstNumberString + this.operator + this.secondNumberString;
+          } else {
+            window.alert("한번에 최대 3자리 수까지 입력할 수 있어요. ");
+          }
```

Comment:
```suggestion
          if (!this.isLessThanThreeDigits(this.secondNumberString)) {
            window.alert("한번에 최대 3자리 수까지 입력할 수 있어요. ");
            return;
          }
          
          this.secondNumberString += digit.textContent;
          this.displayText = this.firstNumberString + this.operator + this.secondNumberString;
          
```

이런식으로 early return 을 해주면 가독성이 한 층 좋아질 것 같습니다.